facefusion/processors/modules/face_editor.py

```diff
@@ -169,7 +169,9 @@ def post_process() -> None:
 def edit_face(target_face: Face, temp_vision_frame : VisionFrame) -> VisionFrame:
 	model_template = get_model_options().get('template')
 	model_size = get_model_options().get('size')
-	crop_vision_frame, affine_matrix = warp_face_by_face_landmark_5(temp_vision_frame, target_face.landmark_set.get('5/68'), model_template, model_size)
+	face_landmark_5 = target_face.landmark_set.get('5/68')
+	face_landmark_5 = (face_landmark_5 - face_landmark_5[2]) * 1.2 + face_landmark_5[2]
```

Comment:
can you explain this 1.2 factor or in general what you are doing here?

Reply:
scaling face-landmark from the center, face_landmark_5[2] is the nose which is considered as center.